src/repology/repology_cli.py

```diff
@@ -174,20 +182,20 @@ def _sbom_fields(self):
         self.df.drop("name", axis=1, inplace=True)
 
     def _get_resp(self, query):
-        LOG.info("GET: %s", query)
+        LOG.debug("GET: %s", query)
         resp = self.session.get(query, headers=self.headers)
         LOG.debug("resp.status_code: %s", resp.status_code)
         if resp.status_code == 404:
             LOG.fatal("No matching packages found")
-            sys.exit(1)
+            raise repology.exceptions.RepologyNoMatchingPackages
         resp.raise_for_status()
         return resp
 
-    def _report(self, args):
+    def _report(self, args, console_report=True):
         """Generate result report to console and to csv file"""
         if self.df.empty:
-            LOG.warning("No matching packages found")
-            sys.exit(1)
+            LOG.debug("No matching packages found")
+            return
```

Comment:
The error path here is unclear to me, this used to become a nonzero exit code. Did you mean to raise an exception here, or is this dealt with elsewhere?